### PR TITLE
Add intro and example to iBeacon Tracker

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Per https://github.com/home-assistant/home-assistant.io/pull/24178#discussion_r976194453

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/78671
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ibeacon.markdown [hunk 1]
```diff
@@ -37,16 +39,53 @@ Consider setting up your iBeacons with a schema similar to the following:
 
 iBeacon devices that do not have stable Major and Minor values are not supported. The system automatically removes iBeacon devices with unstable Major and Minor values once ten (10) or more Major and Minor values have been seen with the same UUID from an iBeacon device with a fixed MAC address.
 
-## Fixed MAC address iBeacons
+## iBeacons with Fixed MAC addresses
```
Comment (bdraco) [inline source/_integrations/ibeacon.markdown]:
```suggestion
## iBeacons with fixed MAC addresses
```

File: source/_integrations/ibeacon.markdown [hunk 2]
```diff
@@ -37,16 +39,53 @@ Consider setting up your iBeacons with a schema similar to the following:
 
 iBeacon devices that do not have stable Major and Minor values are not supported. The system automatically removes iBeacon devices with unstable Major and Minor values once ten (10) or more Major and Minor values have been seen with the same UUID from an iBeacon device with a fixed MAC address.
 
-## Fixed MAC address iBeacons
+## iBeacons with fixed MAC addresses
 
 iBeacons with a fixed MAC address will get their own set of entities for each UUID, major, minor, and MAC address combination, enabling distance and presence detection per physical device basis. In this type of setup, it is permissible to have multiple iBeacons broadcasting the same UUID, Major, and Minor combination as long as you do not exceed five devices.
 
-## Random MAC address iBeacons
+## iBeacon with randomized MAC addresses
```
Comment (bdraco) [inline source/_integrations/ibeacon.markdown]:
```suggestion
## Randomized MAC addresses
```

File: source/_integrations/ibeacon.markdown [hunk 3]
```diff
@@ -37,16 +39,53 @@ Consider setting up your iBeacons with a schema similar to the following:
 
 iBeacon devices that do not have stable Major and Minor values are not supported. The system automatically removes iBeacon devices with unstable Major and Minor values once ten (10) or more Major and Minor values have been seen with the same UUID from an iBeacon device with a fixed MAC address.
 
-## Fixed MAC address iBeacons
+## iBeacons with fixed MAC addresses
```
Comment (bdraco) [inline source/_integrations/ibeacon.markdown]:
```suggestion
## Fixed MAC addresses
```

File: source/_integrations/ibeacon.markdown [hunk 4]
```diff
@@ -37,16 +39,53 @@ Consider setting up your iBeacons with a schema similar to the following:
...
+      minutes: 5
+      seconds: 0
+    above: 20
+condition: []
+action:
+  - service: notify.notify
+    data:
+      message: The black trash can has left the building
+      title: The black trash can has left the building
+mode: single
```
Comment (frenck) [inline source/_integrations/ibeacon.markdown]:
```suggestion
alias: "The black trash can has left the building"
mode: single
trigger:
  - platform: state
    entity_id: sensor.black_trash_bin_estimated_distance
    to: "unavailable"
    for:
      hours: 0
      minutes: 5
      seconds: 0
  - platform: numeric_state
    entity_id: sensor.black_trash_bin_estimated_distance
    for:
      hours: 0
      minutes: 5
      seconds: 0
    above: 20
action:
  - service: notify.notify
    data:
      message: "The black trash can has left the building"
      title: "The black trash can has left the building"
```

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Pull request merged**